actions/lib/base.py

```diff
@@ -38,6 +40,26 @@ def __init__(self, config):
 
         self._session = requests.Session()
 
+    def _reset(self, org):
+        client = Client()
+        gitorgs = client.keys.get_by_name(name='git-orgs', decrypt=True)
+        if gitorgs:
+            dict=json.loads(gitorgs.value)
+        else:
+            dict={}
+        if org.find('@') == -1:
+            org = org + '@' + self.base_url
+        if org in dict:
```

Comment:
If we don't have a record for the org, do we just default to using the config defined for the entire pack?

Reply:
Yes.